controllers/status.go

```diff
@@ -276,12 +270,7 @@ func successfulDeploy(d *appsv1.Deployment) bool {
 		}
 	}
 
-	if d.Status.UnavailableReplicas > 0 {
-		return false
-	}
-
-	return true
```

Comment:
Since there are other early returns here, I think the original form is more readable

Reply:
Will revert this change.